docs/CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ Version 1.9.0 (not released yet)
 - Add level filename to "Level Initializing" console message
 - Properly handle WM_PAINT in dedicated server, may improve performance (DF bug)
 - Fix crash when `verify_level` command is run without a level being loaded
+- Improve handling for players leaving/joining multiplayer games during votes
```

Comment:
I think it is too generic. It may be interesting for people to know what exactly changed (e.g. if they can vote after join or not)

Reply:
I don't really agree - I think the current text is as specific as any player would really care about, but I'm happy to change it to be more specific if you'd like.

Reply:
How is user reading the release notes know what changed if you only say that something "improved"? Improved in what way?

Reply:
Honestly, I think "changelog" and "release notes" are being conflated here - simply saying "improved handling" would be suitable IMO for release notes (directed at a general audience), but a more descriptive listing of specifically what changed would be better for a changelog (directed at folks interested in knowing specifically what changes are made).

I'll rephrase this per your suggestion when I have a chance to loop back to this.